src/browsers.c

```diff
@@ -49,7 +49,7 @@ static char ***browsers_hash = NULL;
 
 /* {"search string", "belongs to"} */
 static const char *browsers[][2] = {
-  /* Game systems: most of them are based of major browsers,
+  /* Game systems: most of them are based off major browsers,
    * thus they need to go before. */
```

Comment:
I think this might be "based on" instead of "based of" or "based off".

The original sentence is:
> Game systems: some of them are Safari-based, thus they need to go before Safari.
> Ref: 0e9dc00 (2015 June)